app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -461,6 +461,16 @@ def fraud_result_pending?(enrollment)
       enrollment.profile&.fraud_pending_reason.present?
   end
 
+  def enrollment_is_enhanced_ipp?(enrollment)
+    IdentityConfig.store.usps_eipp_sponsor_id.eql?(enrollment.sponsor_id)
```

Comment:
I think we want plain `==` here
```suggestion
    IdentityConfig.store.usps_eipp_sponsor_id == enrollment.sponsor_id
```

or honestly, we should add a method to the `InPersonEnrollment` class do you can do

```
enrollment.enhanced_ipp?
```

Reply:
Resolved with commit 9b209900d9b7aa8eeb996673081904c534ab8f69. Added specs in commit 8c474a92b7bf6cd201c9147d1b2b7684fb4f66ed